src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -51,6 +49,8 @@
 @Service
 @Slf4j
 public class EventController {
+
+  private static final String PROMETHEUS = "prometheus";
```

Comment:
```suggestion
  private static final String PROMETHEUS = "prometheus";
  private static final String RHELEMETER= "rhelemeter";
```

Reply:
I used a collection to exclude these two event sources from being logged.